### PR TITLE
Translation for CurseAPIkey massage box

### PR DESCRIPTION
**Pull request by choulily2**
Add i18n Support to CurseForge API Key Message Box

### CODE REVIEW

File: utils/window/main.py [hunk 1]
```diff
@@ -59,12 +59,12 @@ def main(self):
...
-                'Configuration',
-                '请输入 CurseForge API Key',
+                self.factory.language.translate(config.apiKey.title),
+                self.factory.language.translate(config.apiKey.content),
                 show='*',
                 parent=self
             )
-            if result is None:
+            if result is None: # This Doesn't work
                 self.quit()
```
Comment (AnzhiZhang) [inline utils/window/main.py]:
```suggestion
            if result is None:
                return
```

File: lang/zh-cn.yaml [hunk 2]
```diff
@@ -42,3 +42,7 @@ download.logging.downloadFail: |
 download.logging.verifyFail: |
   {0} 个模组校验失败，一般无需手动下载：
   {1}
+config.apiKey.title: 配置
+config.apiKey.content: |
+  请输入Curseforge API key。
+  如果你没有，请参考说明书自己获取，或使用他人的密钥。
```
Comment (AnzhiZhang) [inline lang/zh-cn.yaml]:
```suggestion
  如果你没有，请获取一个或使用他人的密钥
```

File: lang/en-us.yaml [hunk 3]
```diff
@@ -42,3 +42,7 @@ download.logging.downloadFail: |
 download.logging.verifyFail: |
   {0} mods verify failed, generally not manually download required:
   {1}
+config.apiKey.title: configuration
+config.apiKey.content: |
+  Please insert Curseforge API key.
+  If you don't have one.please read the manual to get one.
```
Comment (AnzhiZhang) [inline lang/en-us.yaml]:
```suggestion
config.apiKey.title: Configuration
config.apiKey.content: |
  Please enter a CurseForge API key
  If you don't have one, please get one or use someone's key
```

File: lang/zh-cn.yaml [hunk 4]
```diff
@@ -42,3 +42,7 @@ download.logging.downloadFail: |
 download.logging.verifyFail: |
   {0} 个模组校验失败，一般无需手动下载：
   {1}
+config.apiKey.title: 配置
+config.apiKey.content: |
+  请输入Curseforge API key。
```
Comment (AnzhiZhang) [inline lang/zh-cn.yaml]:
```suggestion
  请输入Curseforge API key
```

**Review by AnzhiZhang — APPROVED** [review]

File: utils/window/main.py [hunk 5]
```diff
@@ -59,12 +59,12 @@ def main(self):
         # ask api key
         if self.factory.config.get('curseForgeAPIKey') == '':
             result = askstring(
-                'Configuration',
-                '请输入 CurseForge API Key',
+                self.factory.language.translate(config.apiKey.title),
+                self.factory.language.translate(config.apiKey.content),
```
Comment (AnzhiZhang) [inline utils/window/main.py]:
```suggestion
                self.factory.language.translate('config.apiKey.title'),
                self.factory.language.translate('config.apiKey.content'),
```

**Review by AnzhiZhang — CHANGES_REQUESTED** [review]

**Comment by choulily2** [comment]
Now Fixed

File: lang/en-us.yaml [hunk 6]
```diff
@@ -42,3 +42,7 @@ download.logging.downloadFail: |
 download.logging.verifyFail: |
   {0} mods verify failed, generally not manually download required:
   {1}
+config.apiKey.title: Configuration
+config.apiKey.content: |
+  Please enter a Curseforge API key
+  If you don't have one,please get one or use someone's key
```
Comment (AnzhiZhang) [inline lang/en-us.yaml]:
```suggestion
  If you don't have one, please get one or use someone's key
```

File: lang/zh-cn.yaml [hunk 7]
```diff
@@ -42,3 +42,7 @@ download.logging.downloadFail: |
 download.logging.verifyFail: |
   {0} 个模组校验失败，一般无需手动下载：
   {1}
+config.apiKey.title: 配置
+config.apiKey.content: |
+  请输入Curseforge API key
+  如果你没有，请获取一个或使用他人的密钥。
```
Comment (AnzhiZhang) [inline lang/zh-cn.yaml]:
```suggestion
  请输入 Curseforge API key
  如果你没有，请获取一个或使用他人的密钥
```

**Comment by choulily2** [comment]
suggests approved

**Pull request merged**